src/components/common/Footer.jsx

```diff
@@ -57,9 +57,11 @@ const Footer = () => {
           <Col md={3} sm={6}>
             <Nav className="flex-column">
               <Nav.Link href="/">Home</Nav.Link>
-              <Nav.Link href="/blog">Blog</Nav.Link>
-              <Nav.Link href="/about">About</Nav.Link>
               <Nav.Link href="/contact">Contact Us</Nav.Link>
+              <Nav.Link href="/about">About</Nav.Link>
+              <Nav.Link href="/psycho-assess">Assessment</Nav.Link>
```

Comment:
i suggest changing this route because usually we don't use psycho as an abbreviation for psychology in english, since psycho implies a crazy person